src/labelling.jsx

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+import React from 'react';
+import classnames from 'classnames';
+
+var getSlidesCount = function (spec) {
+  var slides;
+  slides = Math.ceil(spec.slideCount / spec.slidesToScroll);
+  return slides;
+};
+
+
+export var Labelling = React.createClass({
+
+  render: function () {
+    var template, className, label;
+
+    if (this.props.labellingTemplate) {
+      template = this.props.labellingTemplate;
+    } else {
+      template = (currentSlide, slidesCount) => `${ currentSlide } / ${ slidesCount }`;
+    }
+
+    className = classnames({ 'slick-labelling': true });
+    label = template(
+      this.props.currentSlide + 1, 
+      getSlidesCount({
+        slideCount: this.props.slideCount,
+        slidesToScroll: this.props.slidesToScroll
+      })
+    );
+
+    return (
+      <div className={ this.props.labellingClass ? this.props.labellingClass + ' ' + className : className }>
```

Comment:
Thanks for the deedback @vramana. I'll address the issues and will prepare a new commit.